### PR TITLE
cpansec-minutes-2024-03-13.md: update meeting minutes

### PR DESCRIPTION
**Pull request by timlegge**
@ingydotnet  @perlpunk you are not on the reviewers list but if you have any issues let me know

### CODE REVIEW

**Review by oalders — APPROVED** [review]

File: docs/minutes/cpansec-minutes-2024-03-13.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    - @garu - expressed concern if the CVE numbers go away we would have a problem.
+    - there is no reason we cannot have a local ID to be used (that was somewhat assumed previously - how to manage)
+    - @timlegge (post meeting minute update) if I remember correctly we planned to primarily reference the CVE but did not discount the need for an internal identifier
+    - Deferred to PTS discussion
+6. [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    - …
+7. [CPAN Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+    - @sjn - OpenSSF has quite a bit of info on this topic
+8. [CPAN Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+    - @stigtsp - Public facing Vulnerability Index. - possibly chat at PTC`
```
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-03-13.md:77]: s/PTC/PTS/

File: docs/minutes/cpansec-minutes-2024-03-13.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+   - @sjn - scope is pretty wide - 2FA for pause, TUF (eg), improving security anywhere in opensource.
+   - Trying to schedule a meeting - what kind of things we could/should do and a rough order of magnitude of the work (time, dollars, etc)
+   - @timlegge - to look for a time for a meeting next week
+
+### Ongoing vulnerabilities
+
+- HTTP::Body (marked fixed by mistake) - @stigtsp and Torsten Raudssus (@getty) discussed and a patch has been proposed
+- Session Token Bug @stigtsp discussed with Leon Timmermans (@leont) in January - some issues when passed to C code
+- Two ongoing items are under discussion in closed communication
+- stigtsp and timlegge are working on the old vulnerabilities to have CVEs issued as required
```
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-03-13.md:90]: Would it be meaningful to specify where these old vulnerabilities are found?

**Review by sjn — CHANGES_REQUESTED** [review]
Typo fix needed, and (optionally) check of the source of vulnerabilities that is being reviewed.

**Review by garu — APPROVED** [review]
aside from s/PTC/PTS/ the rest is as I remember it. Great work, @timlegge!

**Review by sjn — APPROVED** [review]
LGTM

**Pull request merged**